CHANGELOG.md

```diff
@@ -4,6 +4,11 @@ All notable changes to this project will be documented in this file.
 The format is based on [Keep a Changelog](http://keepachangelog.com/) 
 and this project adheres to [Semantic Versioning](http://semver.org/).
 
+## [develop]
+
+### Fixed
+- Fix `animate-slide-in-from-bottom` mixin
```

Comment:
This only describes the *actual change* that can be seen in the diff, but does not tell the user what it actually fixed, i.e. what the noticeable implications are. Better to write something more verbose like "Fix `animate-slide-in-from-bottom`  SCSS mixin (fixes missing `VolumeSlider` slide-in animation of `VolumeControlButton` in the legacy skin)"